src/main/groovy/org/transmartproject/core/dataquery/highdim/tworegion/EventValues.groovy

```diff
@@ -0,0 +1,10 @@
+package org.transmartproject.core.dataquery.highdim.tworegion
+
+/**
+ * Represents metadata about one event, possibly linking to multiple junctions
+ */
+interface EventValues {
+    String getCgaType();
+    String getSoapClass();
```

Comment:
Can't these be enums? Not that I can guess what they are since they are not documented.